tests/wpunit/Tribe/CacheTest.php

```diff
@@ -81,27 +75,65 @@ public function it_should_allow_setting_many_different_values_using_array_access
 			'foo8'  => [],
 			'foo9'  => 'false',
 			'foo10' => 'null',
-			'foo11' => false,
-			'foo12' => null,
```

Comment:
`null` and `false` are no longer considered valid values (`false` because of wp_cache_get() and it's definition of false being a not cached return value).

This test was written with the expectation that even though you set `null` and `false` they would be considered valid and set. I think this is problematic for the issues listed above.

This change will adjust our expected behavior for the `Cache` object.

Reply:
The `DateUtils_Test` is failing because it was checking for `false` in the cache. Just one example that would need tweaking if we go this route...

Reply:
See my reply above, I would keep this limit of the `Tribe__Cache` implementation for back-compatibility purposes.